### PR TITLE
added cover letter draft

### PR DESCRIPTION
**Pull request by andrewkern**
Hey @jeromekelleher @petrelharp @dschride -- hoping to get some eyes on this draft of the eLife coverletter. Take a look.

### CODE REVIEW

**Review by petrelharp — APPROVED** [review]
I like it! I'm not thinking of anything to add...

File: elife_coverletter.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Dear Colleagues,
+
+Please consider the attached manuscript entitled, "A community maintained standard library of population genetic
+models" for publication in *elife*. This manuscript describes the first product of the PopSim Consortium, the
```
Comment (dschride) [inline elife_coverletter.md]: should be *eLife*

File: elife_coverletter.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+stdpopsim library, which provides a game-changing resource to the population and statistical genetics community. 
+
+Simulation is a cornerstone tool in population and statistical genetics, wherein genomes or portions thereof
+are simulated for specific genomes under assumed evolutionary and demographic histories. To date researchers in these
```
Comment (dschride) [inline elife_coverletter.md]: "genomes ... are simulated for specific genomes" is a bit maclunkey. Maybe change to "are simulated for specific species/populations."

File: elife_coverletter.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+fields have been forced to implement simulation models independently, leading to duplication of effort and the possibility for error.
+Moreover because of the bespoke nature of such simulations, rigorous benchmarking of inferential procedures against
+an agreed-upon, gold standard has been impossible to date. To rectify this growing issue, we have been actively creating
+a community to build consensus, in the hopes that population and statistical genetics might coalesce around a series of 
+best practices and protocols that center transparency, reproducibility, and inclusiveness. The first output from this community,
+which we have named the PopSim Consortium, is a community-driven open source project, stdpopsim, which provides easy access to a standard catalog of published simulation models from a wide range of organisms and supports multiple simulation engine backends.
```
Comment (dschride) [inline elife_coverletter.md]: maybe replace "simulation engine backends" with "simulators" or "population genetic simulators"?

File: elife_coverletter.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+an agreed-upon, gold standard has been impossible to date. To rectify this growing issue, we have been actively creating
+a community to build consensus, in the hopes that population and statistical genetics might coalesce around a series of 
+best practices and protocols that center transparency, reproducibility, and inclusiveness. The first output from this community,
+which we have named the PopSim Consortium, is a community-driven open source project, stdpopsim, which provides easy access to a standard catalog of published simulation models from a wide range of organisms and supports multiple simulation engine backends. 
+
+In this manuscript we outline the stdpopsim library and give examples of its utility, particularly for benchmarking software
+aimed at performing demographic inference. We believe that this work is a major step forward for the field generally and 
+believe that such open-source, community-driven efforts will lead the future of computational genetics. 
+
+sincerely,
```
Comment (dschride) [inline elife_coverletter.md]: capitalize Sincerely

File: elife_coverletter.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Moreover because of the bespoke nature of such simulations, rigorous benchmarking of inferential procedures against
+an agreed-upon, gold standard has been impossible to date. To rectify this growing issue, we have been actively creating
+a community to build consensus, in the hopes that population and statistical genetics might coalesce around a series of 
+best practices and protocols that center transparency, reproducibility, and inclusiveness. The first output from this community,
+which we have named the PopSim Consortium, is a community-driven open source project, stdpopsim, which provides easy access to a standard catalog of published simulation models from a wide range of organisms and supports multiple simulation engine backends. 
+
+In this manuscript we outline the stdpopsim library and give examples of its utility, particularly for benchmarking software
+aimed at performing demographic inference. We believe that this work is a major step forward for the field generally and 
+believe that such open-source, community-driven efforts will lead the future of computational genetics.
```
Comment (dschride) [inline elife_coverletter.md]: I don't like this "lead the future of computational genetics." Maybe something like "help to modernize computational genetics"? The former is a little too buzzy, while the latter is at least implying a specific point (any field without a standardized benchmarking set is behind the times).

**Review by dschride — COMMENTED** [review]
Looks good. Some minor comments.

File: elife_coverletter.md (hunk 2 above)
Comment (andrewkern) [reply elife_coverletter.md]: lol

File: elife_coverletter.md (hunk 5 above)
Comment (andrewkern) [reply elife_coverletter.md]: changing to "...will modernize computational genetics. "

**Pull request merged**